addons/account/models/account_invoice.py

```diff
@@ -1164,6 +1171,18 @@ class AccountInvoiceTax(models.Model):
     sequence = fields.Integer(help="Gives the sequence order when displaying a list of invoice tax.")
     company_id = fields.Many2one('res.company', string='Company', related='account_id.company_id', store=True, readonly=True)
     currency_id = fields.Many2one('res.currency', related='invoice_id.currency_id', store=True, readonly=True)
+    amount_in_group = fields.Monetary(compute='_compute_amount_in_group')
```

Comment:
About the model's name, 'group' seemed to be a good approach because it's only meant to 'group' the taxes at the reporting level. Type seems more to me like a property of the tax that would impact its behavior. But then again, it's just my opinion, so I'll let you decide about it.